src/smolagents/models.py

```diff
@@ -686,6 +691,12 @@ def __call__(
     ) -> ChatMessage:
         import litellm
 
+        # IMPORTANT - Set this to TRUE to add the function to the prompt for Non OpenAI LLMs
+        if litellm.supports_function_calling(model=self.model_id) == True:
```

Comment:
```py
if litellm.supports_function_calling(model=self.model_id):
```
Or
```py
if litellm.supports_function_calling(model=self.model_id) is True:
```
if you want to be very specific.

Reply:
I'm new to this project so I'm not sure if some idioms are acceptable or not.

What do you think of:
`litellm.add_function_to_prompt = not litellm.supports_function_calling(model=self.model_id)`